Signal/src/ViewControllers/ConversationView/Cells/OWSBubbleView.m

```diff
@@ -161,19 +161,52 @@ + (UIBezierPath *)maskPathForSize:(CGSize)size
     CGFloat bottomRounding
         = (useSmallCorners_Bottom ? kOWSMessageCellCornerRadius_Small : kOWSMessageCellCornerRadius_Large);
 
+    const CGFloat topAngle = 3.0f * M_PI / 2.0f;
+    const CGFloat rightAngle = 0.0f;
+    const CGFloat bottomAngle = M_PI / 2.0f;
+    const CGFloat leftAngle = M_PI;
+
     [bezierPath moveToPoint:CGPointMake(bubbleLeft + topRounding, bubbleTop)];
+
+    // top line
     [bezierPath addLineToPoint:CGPointMake(bubbleRight - topRounding, bubbleTop)];
```

Comment:
I wonder if the `addLineToPoint` calls are now redundant, e.g. whether `UIBezierPath` will "close" the shape between the arcs.

Reply:
Hmm.. it appears that it does. I've made this change.